### PR TITLE
Add information about template data

### PR DESCRIPTION
**Pull request by fritzmg**

### CODE REVIEW

File: docs/dev/framework/templates.md [hunk 1]
```diff
@@ -138,3 +138,37 @@ The output of the `image.html5` template will be:
...
+You can inspect the available template data by using either
+
+```php
+<?php $this->dumpTemplateVars() ?>
+```
+
+or
+
+```php
+<?php dump($this) ?>
```
Comment (xchs) [inline docs/dev/framework/templates.md:161]:
```suggestion
<?php dump($this); ?>
```

File: docs/dev/framework/templates.md [hunk 2]
```diff
@@ -138,3 +138,37 @@ The output of the `image.html5` template will be:
...
+content element templates or element templates of modules will have their complete
+data record available in the template. For example, in any content element template,
+the complete data set of the database record of the element will be available in 
+the template via `$this->…`. The content of some fields might have been changed 
+by the content element or module controller though.
+
+You can inspect the available template data by using either
+
+```php
+<?php $this->dumpTemplateVars() ?>
```
Comment (xchs) [inline docs/dev/framework/templates.md:155]:
```suggestion
<?php $this->dumpTemplateVars(); ?>
```

**Review by xchs — CHANGES_REQUESTED** [review]

File: docs/dev/framework/templates.md (hunk 1 above)
Comment (fritzmg) [reply docs/dev/framework/templates.md:161]: `;` is not necessary in this case. Personally, I prefer it without `;` :)

Comment (xchs) [reply docs/dev/framework/templates.md:161]: Ok. 👍

**Review by Toflar — APPROVED** [review]

**Pull request merged**